utils/eudev/Makefile

```diff
@@ -132,6 +132,8 @@ define Package/eudev/install
 			80-net-name-slot.rules) \
 		$(1)/lib/udev/rules.d
 
+	$(INSTALL_DIR) $(1)/run
```

Comment:
I don't get this.  `Package/.../install` is run on the build host at build time.

`/run/` is a directory that exists on the target at boot-time.

What is happening at install-time that would depend on this?

Reply:
On LEDE there's no `/run/` directory at boot-time, it was removed. So packages using this directory fail.

Reply:
Still don't understand.  How do they fail?  Nothing should be happening in this directory at build or install time.

Reply:
Okay, Stijn explained it to me.  You just want it created here because you'll need it at run-time.

Top-level directories are created in `base-files`, as far as I know.  This would happen there... which makes more sense, because you don't want multiple packages that use `/run` all attempting to create it.  But everyone needs `base-files`.

Reply:
Ok, sorry for not being clear. That's the case, I need it at run-time so I create it in build time.

I think this should be addressed creating the `/run` directory at LEDE level as a tmpfs partition as @stintel suggests. As I understand, it makes `/run` available earlier at boot time so eudev, for example, can use it (now, with the workaround I proposed, eudev fails the first times but when it retries it founds `/run` and works as expected).